src/date-picker/date-picker.jsx

```diff
@@ -149,11 +149,14 @@ const DatePicker = React.createClass({
 
   getDefaultProps() {
     return {
-      formatDate: DateTime.format,
+      formatDate: (date) => {
```

Comment:
That's pretty much what I did (sorry, went googling before I saw your comment), and while it works in the console, I'm still seeing the full function in the docs...